src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -475,6 +496,7 @@ ScriptEditorWidgetColorOptions::ScriptEditorWidgetColorOptions(QWidget * pParent
 	gbox->setInsideSpacing(0);
 
 	addColorSelector(gbox, __tr2qs_ctx("Background:", "editor"), &g_clrBackground, true);
+	addColorSelector(gbox, __tr2qs_ctx("Background Highlight:", "editor"), &g_clrBackgroundHighlight, true);
```

Comment:
> Maybe you'd prefer "Highlight background"?

that would be a way to go, but which ever way you wanna go multiline or not but ide make it consistent with the code part cus atm you have a mix that are neither here nor there.

something like https://github.com/IceN9ne/KVIrc/pull/1 for however you want to go